backend/log/log.go

```diff
@@ -0,0 +1,54 @@
+package log
```

Comment:
will be moved to backend package

Reply:
Decided to keep the interface in a separate package. But a `Logger` variable is now available in the `backend` package.